<?xml version="1.0" encoding="UTF-8"?><testrun name="OracleConnectionTest" project="koing" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testsuite name="koing.kosta180.controller.OracleConnectionTest" time="1.127">
    <testcase name="testConnect" classname="koing.kosta180.controller.OracleConnectionTest" time="1.127">
      <error>java.sql.SQLException: ORA-01017: invalid username/password; logon denied
&#13;
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:440)&#13;
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:389)&#13;
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:382)&#13;
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:573)&#13;
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:431)&#13;
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:445)&#13;
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:191)&#13;
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:366)&#13;
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:752)&#13;
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:366)&#13;
	at oracle.jdbc.driver.PhysicalConnection.&lt;init&gt;(PhysicalConnection.java:536)&#13;
	at oracle.jdbc.driver.T4CConnection.&lt;init&gt;(T4CConnection.java:228)&#13;
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:32)&#13;
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:521)&#13;
	at java.sql.DriverManager.getConnection(Unknown Source)&#13;
	at java.sql.DriverManager.getConnection(Unknown Source)&#13;
	at koing.kosta180.controller.OracleConnectionTest.testConnect(OracleConnectionTest.java:21)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
